code/modules/spells/spell_types/undirected/conjure_item/briar_claw.dm

```diff
@@ -0,0 +1,56 @@
+/datum/action/cooldown/spell/undirected/conjure_item/briar_claw
+	name = "Briar Claw"
+	desc = "Turns one hand into a wolf's claw."
+	button_icon_state = "dendor"
+	invocation = "Beast-Lord, lend me the claws of a volf."
+	invocation_type = INVOCATION_WHISPER
+	spell_type = SPELL_MIRACLE
+	antimagic_flags = MAGIC_RESISTANCE_HOLY
+	associated_skill = /datum/skill/magic/holy
+	required_items = list(/obj/item/clothing/neck/psycross/silver/dendor)
+	spell_cost = 15
+	item_duration = 1 MINUTES
+	cooldown_time = 4 MINUTES
+	item_type = /obj/item/weapon/briar_claw/left
+	uses_component = TRUE
+	refresh_count = 0
+	attunements = list(
+		/datum/attunement/blood = 0.3,
+		/datum/attunement/earth = 0.7
+	)
+
+/datum/action/cooldown/spell/undirected/conjure_item/briar_claw/make_item()
```

Comment:
you should select the hand in before_cast and confirm the user is a carbon in can_cast_spell